theunderground/movies.py

```diff
@@ -70,8 +79,9 @@ def add_movie():
                     category_id=form.category.data,
                     length=length,
                     aspect=True,
-                    genre=0,
+                    genre=genre,
                     sp_page_id=0,
+                    ds_mov_id=ds_mov_id,
```

Comment:
Remove this line. After the declaration of `db_movie`, do 
```py
db_movie.ds_mov_id = db_movie.movie_id
```

Reply:
resolved in 46a2678